packages/design-system/Component-Migration-Checklist.md

```diff
@@ -0,0 +1,33 @@
+# Steps to migrate components
+
+- [ ]  Pick the component to be migrated. Mark it as in progress (highlight in yellow) on the migration sheet [Component Migration planning](https://docs.google.com/spreadsheets/d/1pcIMvGwknhbjnG1yxKH2UnRIOPwW5lw_xiFxrJdb1E8/edit?usp=drivesdk)
+- [ ]  Create an issue for it on [appsmith/ce](http://github.com/appsmithorg/appsmith). Announce that you’re doing this on #design-system and #team-tech channel.
+    - [ ] All PRs should have an issue.
+    - [ ] All issues should have estimates.
+    - [ ] All migration-related issue goes inside the migration epic
+    - [ ] All migration issues should have labels ads migration
+    - [ ] All design system issues should have a design system pod label. Then, zenhub can track the issues.
+    - [ ] All issues should have sprint added to it.
+- [ ]  Copy the files over to the [appsmith/design-system](http://github.com/appsmithorg/design-system/) repository.
+      - [ ]  Copy any unit test files over as well if they exist.
+      - [ ]  In the files, props.theme variables become variables used from variables.css
+      - [ ]  component-specific variables and tokens go in index.css (make sure you order this alphabetically)
+      - [ ]  Add stories for every state and variation of the component
+      - [ ]  Make sure to add the exports of the components you’ve added in the src/index.ts file
+- [ ]  Create a branch to change the imports on appsmith/ce.
+      - [ ]  On this branch, delete the files for the component you’ve just migrated.
+      - [ ]  Everywhere the component was used, change the import to point to the design-system package instead.
+      - [ ] If you've migrated a component that is used a specific new places in the ee-repo, make updates there as well.
```

Comment:
@tanvibhakta : Also, add a mandatory step to test locally on both ce & ee. Then, the developer will know is that breaking anything on ee. Then, he needs to create a branch on ee and do import changes.